src/SwaggerProvider.DesignTime/v3/DefinitionCompiler.fs

```diff
@@ -194,11 +193,11 @@ type DefinitionCompiler(schema: OpenApiDocument, provideNullable) as this =
                 ty,
                 getterCode =
                     (function
-                    | [ this ] -> Expr.FieldGetUnchecked(this, providedField)
+                    | [ this ] -> Expr.FieldGet(this, providedField)
```

Comment:
Please explain this change as well.

Why we should not use Expr methods crafted for TP authors as part of TP SDK?
https://github.com/fsprojects/FSharp.TypeProviders.SDK/blob/master/src/ProvidedTypes.fsi#L536-L539

It is probably because you SDK out of assembly and have no longer assess to them... Hmm